systemvm/debian/opt/cloud/bin/setup/postinit.sh

```diff
@@ -33,6 +42,10 @@ then
   fi
 fi
 
+if [ "$TYPE" == "CKSNode" ]; then
+  pkill -9 dhclient
```

Comment:
Note - if we've guarded how to call/run the dhclient, we may not need to do this.